pollination/honeybee_radiance/glare.py

```diff
@@ -0,0 +1,122 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+
+
+@dataclass
+class DCGlareGlareAutonomy(Function):
+    """Calculates glare autonomy. The glare autonomy is the fraction of hours (occupied) 
+    without any detected glare. The detection of glare is controlled by glare_limit."""
+
+    dc_direct = Inputs.file(
+        description='Path to dcdirect.', path='dc_direct.mtx',
+    )
+
+    dc_total = Inputs.file(
+        description='Path to dctotal.', path='dc_total.mtx',
+    )
+
+    sky_vector = Inputs.file(
+        description='Path to sky vector.', path='sky.smx'
+    )
+
+    view_rays = Inputs.file(
+        description='Path to view ray.', path='view_rays.ray',
+    )
+
+    glare_limit = Inputs.float(
+        description='Glare limit indicating presence of glare.', default=0.4
+    )
+
+    threshold_factor = Inputs.float(
+        description='Constant threshold factor in cd/m2.', default=2000
+    )
+
+    occupancy_scheulde = Inputs.file(
+        description='Path to occupancy schedule.', path='occupancy_schedule.csv',
+    )
+
+    @command
+    def run_dcglare(self):
+        return 'honeybee-radiance dcglare two-phase dc_direct.mtx dc_total.mtx ' \
+            'sky.smx view_rays.ray --glare-limit {{self.glare_limit}} ' \
+            '--threshold-factor "{{self.threshold_factor}}" --occupancy-schedule ' \
+            '{{self.occupancy_schedule}} --output occupied.ga'
+
+    dcglare_glare_autonomy = Outputs.file(
+        description='Output dcglare glare autonomy file.', path='occupied.ga'
+    )
+
+
+@dataclass
+class DCGlare(Function):
+    """Calculates DGP for all sky conditions in the sky matrix."""
```

Comment:
I don't know if the name DCGlare tells me what this Function really does but if it makes sense to you then it is fine with me.